test/e2e/e2e-test.ts

```diff
@@ -0,0 +1,46 @@
+import { Selector } from 'testcafe';
+import nock from 'nock';
+import { App, Config } from '../../';
+import { nockRequests } from '../util/nocks';
+
+const configInstance = new Config();
+const config = configInstance.config();
+
+const rootPath = `http://${config.host}:${config.port}`;
+
+let app: App;
+
+const userPresences = nockRequests(config);
+
+fixture `Client`
+    .page `${rootPath}`
+    .before(async () => {
+        app = await App.create({ port: config.port, host: config.host });
+    })
+    .after(async () => {
+        await app.closeServer();
+    });
+
+test('Polls repeatedly', async t => {
+    userPresences.persist(false);
```

Comment:
We should assert the initial state of the page to be sure that our test data is correct and we don't have the updated data from the start.